### ISSUE TITLE
No way to get the user id of the player

### ISSUE THREAD

**Issue by TheSecondReal0** [samsface/godot-discord-game-sdk#6]
There seems to be no way to see what the discord user ID of the player is, I need this to be able to differentiate stuff effectively.

This seems to be implemented in the UserManager part of the Discord SDK in GetCurrentUser()

**Comment by samsface** [comment]
Yeah I only implemented what I could during the jam. You want to just fork this and I can help you get it compiling so can just add the stuff yourself?

**Comment by TheSecondReal0** [comment]
sure, I don't have time to do that right now, busy with uni and job/internship searching. For now I found a workaround that's kinda really janky and it's just figuring out your user id by sending a lobby message to yourself? It's kinda weird but I think it works.

**Comment by samsface** [comment]
> sure, I don't have time to do that right now, busy with uni and job/internship searching. For now I found a workaround that's kinda really janky and it's just figuring out your user id by sending a lobby message to yourself? It's kinda weird but I think it works.

Haha that's genius. Honestly don't have time to implement the whole networking layer they provide. But if you only need like 2 or 3 more calls to finish your game I'm happy to do it.
